### PR TITLE
Pagination php example

### PR DESCRIPTION
**Pull request by sandervanhooft**
#410

Code block selector fixed in #426 by @albert-carreras .

### CODE REVIEW

**Comment by sandervanhooft** [comment]
@albert-carreras 

This issue seems familiar to the one that you fixed here (enable profile method):

- https://github.com/mollie/api-documentation/commit/cea27849f3ec428a9a0d90270359d293b46c760c
- https://github.com/mollie/api-documentation/commit/43883d6bc0b1eb639b1934cef40c44de28341cfd

I have no idea how this build is being processed or how the code block selector works, but it seems `link`/`id` related.

Perhaps we could circumvent the issue by adding a prefix like @mvdpanne does [here](https://github.com/mollie/api-documentation/pull/422#issue-255316032):

```
* - .. param-name:: billingEmail
       :prefix: bankTransfer
```

File: source/guides/pagination.rst [hunk 1]
```diff
@@ -70,11 +70,33 @@ Example of v2 pagination
```
Comment (albert-carreras) [inline source/guides/pagination.rst]: For the code-block-selector to work, the title needs to be consistent with the rest of examples.
Like this:

```
Example
-------

.. code-block-selector::
```

Comment (sandervanhooft) [reply source/guides/pagination.rst]: I understand, but what if we want to have a more distinguishable title? Is this possible?

V1 vs V2 is an important difference, should not be overlooked.

Comment (albert-carreras) [reply source/guides/pagination.rst]: Hmm I agree!

https://github.com/mollie/api-documentation/pull/426

**Comment by vernondegoede** [comment]
@sandervanhooft I believe this one can be merged.

**Comment by sandervanhooft** [comment]
@vernondegoede Reverted `package-lock.json`.

**Pull request merged**